packages/pn-personafisica-webapp/src/pages/NuovaDelega.page.tsx

```diff
@@ -395,7 +457,23 @@ const NuovaDelega = () => {
                         <VerificationCodeComponent code={values.verificationCode} />
                       </Stack>
                       <Divider sx={{ marginTop: '1rem' }} />
-                      <Grid container sx={{ marginTop: '1rem' }}>
+                      <Stack
+                        sx={{ mt: '1rem' }}
+                        alignItems="flex-start"
+                        justifyContent={'flex-start'}
+                      >
+                        <Stack>
+                          <Button
+                            sx={{ marginTop: '1rem', margin: 'auto' }}
+                            type={'submit'}
+                            variant={'contained'}
+                            data-testid="createButton"
+                          >
+                            {t('nuovaDelega.form.submit')}
+                          </Button>
+                        </Stack>
+                      </Stack>
+                      {/* <Grid container sx={{ marginTop: '1rem' }}>
```

Comment:
wouldn't be better to just erase this part, instead of keeping it commented out?